arkane/encorr/bac.py

```diff
@@ -567,13 +567,14 @@ def _get_mol_coeff(self, mol: Molecule, multiplicity: int = 1) -> float:
             self._k_coeffs[mol.id] = coeff
         return coeff
 
-    def fit(self, db_names: Union[str, List[str]] = 'main', **kwargs):
+    def fit(self, weighted: bool = False, db_names: Union[str, List[str]] = 'main', **kwargs):
```

Comment:
Just to be clear, our decision is that unweighted fittings is the preferred fitting by default?

Reply:
I think so, but could change depending on what we discover in the future.